src/configs/env.ts

```diff
@@ -50,6 +50,10 @@ export const SUPERSET_JURISDICTIONS_DATA_SLICE =
   process.env.REACT_APP_SUPERSET_JURISDICTIONS_DATA_SLICE || '0';
 export type SUPERSET_JURISDICTIONS_DATA_SLICE = typeof SUPERSET_JURISDICTIONS_DATA_SLICE;
 
+export const SUPERSET_IRS_REPORTING_JURISDICTIONS_DATA_SLICE =
+  process.env.REACT_APP_SUPERSET_IRS_REPORTING_JURISDICTIONS_DATA_SLICE || '0';
```

Comment:
New env variable to specify the superset slice @moshthepitt put together

Reply:
@cKellyDesign could we also add this to the sample file?  Also, please share the current value on #reveal or somewhere?

Reply:
added